packages/cap-config-writer/src/cap-writer/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { t } from '../i18n';
+import type { CapRuntime } from '@sap-ux/odata-service-inquirer';
+
+/**
+ * Generates a launch text for the launching of applications.
+ *
+ * @param capType The type of CAP runtime used in the application will be provided for CAP applications.
+ * @param projectName The project's name, which is the module name.
+ * @param appId If appId is provided, it will be used to open the application instead of the project name. This option is available for use with npm workspaces.
+ * @returns The launch text for the application.
+ */
+export function getLaunchText(capType: CapRuntime, projectName: string, appId?: string): string {
```

Comment:
Is this relevant for the this PR?

Reply:
We need `getLaunchText` to be exported from `cap-config-writers` to generate launch text in the README for CAP files.

Reply:
Yes but thats not the concern of the read me writer functionality as that is a provided property. It is not called directly from the read me writing?

Reply:
No - i am exporting to be made available to the caller of the generate API.

Reply:
moved to another [PR](https://github.com/SAP/open-ux-tools/pull/2043)